html/gallery_section.html

```diff
@@ -0,0 +1,182 @@
+<section id='gallery_section'>
+  <span id='gallery_anchor' class='anchors'></span>
+  <h2>Gallery</h2>
+  <div class='gallery_thumbnails'>
+    <h3>Videos</h3>
+    <div class="slider gallery_responsive">
+      <div>

```

Comment:
for the gallery you don't need a separate section for the thumbs and the iframes lightbox effect.
you should be able to build a unordered list with a list of links when on click
opens a lightbox.

```suggestion
     <ul>
      <li><a href="https://www.youtube.com/watch?v=HbYFY_lx6pY"><img src="img/videos_img01.jpg" alt=""></a></li>
      <li><a href="https://www.youtube.com/watch?v=HbYFY_lx6pY"><img src="img/videos_img01.jpg" alt=""></a></li>
      <li><a href="https://www.youtube.com/watch?v=HbYFY_lx6pY"><img src="img/videos_img01.jpg" alt=""></a></li>
      <li><a href="https://www.youtube.com/watch?v=HbYFY_lx6pY"><img src="img/videos_img01.jpg" alt=""></a></li>
      <li><a href="https://www.youtube.com/watch?v=HbYFY_lx6pY"><img src="img/videos_img01.jpg" alt=""></a></li>
    </ul>

```